utils/shared.py

```diff
@@ -50,6 +71,18 @@ def make_blast_db(cm_library):
     runner.run(cmd)
 
 
+def verify_ssi_exists(cm_library):
+    """Take a folder with covariance models and check if the ssi file exists.
+    The ssi file is only needed for all.cm and not for individual cm files.
+    """
+    all_cm = Path(cm_library) / "all.cm"
+    if not all_cm.exists():
```

Comment:
I'm a bit confused with this logic and the comment above. The comment says:

> The ssi file is only needed for all.cm

However, the method here does nothing if `all.am` exists. My understanding of the comment above would be to do exactly the opposite.